apps/web-giddh/src/assets/css/color.scss

```diff
@@ -68,172 +59,56 @@ $cinnabar-orange: #df4927;
-$dim-gray-new: #616161;
-$ghost-white: #fdfdfd;
-$nevada: #717274;
-$bright-red: #c63b13;
-$autumn-fog: #bbb;
-$gray-98: #fafafa;
-$orange-peel: #ffa000;
-$aluminum: #8f9091;
-$gray-75: #bfbfbf;
-$heather: #b3b4b5;
-$vivid-orange: #edaf12;
-$violet-red: #f74074;
-$java: #27a264;
-$gray-44: #707070;
-$comet: #6b6c72;
-$ghost: #d0d0d3;
-$dark-green: #229e2b;
-$electric-blue: #0c8fe6;
-$orange-red: #ff4500;
-$shamrock: #36a1ce;
+$light-purple: #b590ff;
+$faux-cadet-blue: #adb4c3;
+$hint-of-cyan: #19c2d0;
 $skin-tone: #ffceb1;
-$warning: #d58512;
-$granite-gray: #636363;
-$crimson: #e34818;
-$light-grayish: #7889a1;
-$san-juan: #48565f;
-$bud-green: #79bd58;
-$eastern-blue: #0e909a;
-$cloud-burst: #202e5a;
-$atlantis: #96bc2d;
-$dell: #2a651d;
-$sherpa-blue: #004254;
-$dark-orange: #ff8c00;
-$burgundy: #82001d;
-$dull-pink: #6b1438;
-$tickle-pink: #f4749b;
+$grayish-blue: #7889a1;
 $deep-pink: #950069;
 $dark-magenta: #542852;
-$jade-color: #00b177;
-$grimmys-gray: #777777;
-$navy-blue: #025cff;
-$faux-cadet-blue: #adb4c3;
-$gray-black: #222222;
-$lochmara: #2a6496;
-$hint-of-red: #f5f4f3;
-$light-red: #ea795b;
-$dark-red: #d64830;
-$vivid-blue: #2799fb;
-$gray-x11: #bfbebe;
-$light-silver-gray: #c4c0c0;
-$hint-of-cyan: #19c2d0;
-$dark-navy-blue: #0467a8;
-$bright-orange: #f6b232;
-$deep-green: #28ab00;
-$extra-dark-gray: #afabab;
-$cobalt-blue: #2262d1;
-$extra-bright-red: #f80606;
-$dark-pink-swan: #b7b7b7;
-$cerulean: #10aae0;
-$light-white-smoke: #f1f1f1;
-$dark-blue: #6b73c8;
-$quick-silver: #a0a0a0;
-$strong-red: #cf0606;
-$extra-pink-swan: #b3b3b3;
-$bright-gray: #eceeef;
-$cinder: #292b2c;
-$ghost-white-light: #f7f7f9;
-$mid-gray: #636c72;
-$dark-nero: #262626;
-$gray-shade: #858484;
-$ford-gray: #979797;
-$dark-white-smoke: #efefef;
-$light-pink-swan: #b4afaf;
-$silver-shade: #c1c1c1;
-$blue-shade: #0fa6e9;
-$dark-silver: #bebebe;
-$strong-green: #63b214;
+$burgundy: #82001d;
+$dull-pink: #6b1438;
+$sherpa-blue: #004254;
+$atlantis: #96bc2d;
+$cloud-burst: #202e5a;
+$bud-green: #79bd58;
+$deep-green-cyan: #125446;
+$granny-apple: #c9e6ca;
+$chablis: #ffece094;
 $valhalla: #2c2c43;
 $medium-spring-blue: #11b1e5;
 $light-maroon: #de4d3c;
 $medium-black-cyan: #0e78aa;
-$dull-orange: #db7f53;
 $russian-black: #0f0f27;
+$pale-rose: #ebccd1;
 $floral-white: #fff8eb;
-$royal-blue: #0c5fc1;
-$red-shade: #e33536;
-$black-pearl: #1d1e1f;
-$lavender-blue: #bdc3ff;
-$extra-silver: #bdbaba;
-$pale-red: #fbc2a8;
-$light-red: #f15757;
-$cerebellum-gray: #cbcbcb;
-$gray-49: #7d7d7d;
-$gray-nobel: #9a9a9a;
-$extra-light-gray: #c7c7c7;
-$gray-68: #adadad;
-$gray-87: #dedede;
-$light-gray-extra: #e2e2e2;
-$leave-green: #089328;
-$medium-gray: #a2a2a2;
-$gray-82: #d1d1d1;
-$deep-green-cyan: #125446;
-$granny-apple: #c9e6ca;
-$spun-pearl: #b1b1b3;
-$gray-red: #9b9b9b;
-$dark-gray-red: #2c2c2c;
-$extra-dark-orange: #e34b26;
-$selago: #fefdfe;
-$bright-grey: #525265;
-$light-grayish-navy-blue: #73737f;
-$valspar-paint-autumn-fog: #bbbbbb;
+$cactus: #587058;
```

Comment:
I have seen some of color var added only on color.scss file and not using on other css file. find Below to name of color var and just make sure if any of the color is not using on css then please remove that color on color.scss file.

$burgundy,  $dull-pink, $atlantis , $violet-red, $java, $bus-yellow , $cactus, $cornflower-blue-dark, $pale-rose, $cloud-burst, $deep-green-cyan, $granny-apple, $chablis

Reply:
@shalinee-walkover  These colors are added in tb-pl-bs, onboarding and in design filters.